### ISSUE TITLE
Search to narrow meeting list by day, type, etc, requires browser refresh.

### ISSUE THREAD

**Issue by OIG-Webmaster** [code4recovery/12-step-meeting-list#222]
**Describe the bug**

It was brought to our attention that after executing a search from the list format only a subset of the meetings meeting the criteria is shown. In order to see the full list of filtered meetings, a browser screen refresh must be executed. This occurs using both Chrome and Safari on my Big Sur Mac. I do not know the type of computer or browser belonging to the person who reported this to me.

**To Reproduce**
1. Go to https://oahucentraloffice.com/meetings/
2. Click on a choice in the search menu. Found by changing the day from the current day to "any day" and then back again. Generally, only a subset of meetings are shown. A screen refresh will show all expected meetings.

**Expected behavior**
I expect to see the full list of expected meetings for the day or location instead of a subset.

**Screenshots**

**Desktop (please complete the following information):**
 - OS: Big Sur
 - Browser Chrome/Safari/Firefox/Edge
 - Version  87.0.4280.63 /14.0.1 (16610.2.11.51.8)/82.0.3/86.0.622.69 

Problem before refresh:
![image](https://user-images.githubusercontent.com/65614651/99352268-6c04ae80-2846-11eb-93e6-de3833e695b2.png)

After refresh:
![image](https://user-images.githubusercontent.com/65614651/99352402-a53d1e80-2846-11eb-981d-0c7464aec2c2.png)

Mahalo!

**Comment by tech2serve** [comment]
I can confirm this is happening with your site. I think we need to look at a couple more data points.

Please provide version information of:
1. Wordpress
1. TSML

Do you have the Enable jQuery Migrate Helper plugin installed? This *may* be required if you've updated to Wordpress 5.5 or later. 

I'm seeing an error in your console which makes me think this is a jQuery issue, but the root cause isn't clear. I have not seen this error before. If you have a staging or test site, can you try disabling other plugins and seeing if the issue resolves?

**Comment by OIG-Webmaster** [comment]
Aloha,

We are running WordPress 5.5.3 using the Twenty-Fourteen theme.
Our TSML version is 3.9.3. I've to turn off auto-updating for this as our site (along with others) went down on a couple of earlier updates. 

We do not have the JQuery Migrate Helper plugin installed. I will do so now. 

We do not have a staging/test site at this time.

I'll install the JQuery plugin and report back. 

Mahalo

**Comment by OIG-Webmaster** [comment]
I installed the JQuery Migrate Helper plugin and it immediately found logged this on the meeting list page: 


2020-11-17 08:20:24 | https://oahucentraloffice.com/wp-content/themes/twentyfourteen/js/functions.js: jQuery.fn.load() is deprecated
-- | --

**Comment by tech2serve** [comment]
If there is an update for the 2014 theme, you should apply it. If not, you may need to contact them and ask the status. I can't say for certain this will solve the Javascript issue with loading filtered meetings, but I'd venture to guess there is a better than 50-50 chance it will. My suggestion is to start with that, and test again. Please report back.

You may also see warnings when you go to edit a meeting. You can ignore those for now.

**Comment by OIG-Webmaster** [comment]
Currently, the version of the theme is the most up to date, last modified in August I believe. I'll let them know. 

Mahalo!

**Comment by tech2serve** [comment]
You're welcome. 

I'm going to close this Issue but we can easily re-open it if the theme isn't the problem. 

Either way, please keep me posted. We are experiencing more issues since Wordpress started their transition away from deprecated jQuery and jQuery Migrate. I suspect we'll be slogging through these strange anomalies for the next few months.

**Comment by OIG-Webmaster** [comment]
Just so you know, I switched briefly to a theme that did not use jquery and the problem persisted. 

Mahalo

**Comment by tech2serve** [comment]
Was the error still present in the javascript console?

**Comment by tech2serve** [comment]
I tried to ingest your meeting feed into my dev site, but I get these alerts and no meetings. 

![image](https://user-images.githubusercontent.com/31904316/99453750-0ae2e680-28eb-11eb-81ea-d34794fd578f.png)

Do you have a couple of meetings that you somehow forced to not have locations with them?

**Comment by OIG-Webmaster** [comment]
I'll take a look, they should all have at least general locations. I'm not the one who loads the meetings regularly, so it's possible one of the volunteers made an error.

**Comment by OIG-Webmaster** [comment]
No clue. I pulled the CSV file and can't find any mention of Meeting Location by Appointment. I guess I don't know what forced not to have locations means either, all the meetings appear to have some location.  We did set our online only meetings up as location = online only and the address to simply Honolulu, HI.

**Comment by tech2serve** [comment]
How you handled the online should be fine. 

I'm assuming you didn't mean to close this issue?

**Comment by OIG-Webmaster** [comment]
Okay, I've somehow fixed the problem. I don't know about why you couldn't load our feed, but I can't find any meetings without locations. However, here's what I discovered. 

1. We had two custom meeting types created in functions.php dealing with COVID-19 status. We told everyone not to use them after realizing they weren't terribly helpful once we closed all meetings except online down. Unfortunately, someone used them again in recently added meetings. I simply removed the code (which was in our child theme). I don't know if they caused some conflict. 

2. A group of newly added meetings is part of a group, but whoever entered one of them used a different address, a digit or two off. I fixed that so they all show the same address. 

3. I took a look at the meeting on Monday that was the beginning of the cutoff point. I could see no errors, but I retyped the address (with the same thing), and then saved it. 

The list seems to be working properly again.
